core/domain/entities/routing/handlers/admin/AdminRoute.php

```diff
@@ -62,7 +62,7 @@ public function __construct(
      */
     public function matchesCurrentRequest()
     {
-        return $this->request->isAdmin();
+        return $this->request->isAdmin() || $this->request->isActivation();
```

Comment:
this route needs to load during regular admin routes as well as activation requests so that our EE admin menu shows up on the first request after activation